charts/homebridge/templates/deployment.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
     {{- include "homebridge.labels" . | nindent 4 }}
 spec:
   replicas: {{ .Values.replicaCount }}
+  strategy:
+    type: {{ .Values.strategyType }}
```

Comment:
This is missing in the `values.yaml` file.  Can you please add it there as well with a default value (probably `Recreate` ?)

Reply:
Done!